content/birds/carl.md

```diff
@@ -10,7 +10,7 @@ social:
   github: carljm
   stackoverflow: 3207
 summary: |
-  Carl is a co-founder
+  Carl (he/him) is a co-founder
```

Comment:
@carljm I took these from your Twitter, but let me know if you prefer something else.